### PR TITLE
Add a helper script to automate the TUF updates.

### PR DESCRIPTION
**Pull request by ivanalejandro0**
**DO NOT MERGE THIS**

If the code is ok after review we should integrate this work into the `bitmask_bundler` repository.
I'm making a pull request here since that repo is empty until we do some push.

### CODE REVIEW

File: tuf-stuff.sh [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+    if [[ $ARCH == "64" ]]; then
+        TUF_ARCH='linux-x86_64'
+    else
+        TUF_ARCH='linux-i386'
+    fi
+
+    # Download old repo metadata
+    echo "${cc_yellow}-> Downloading metadata files from the old bundle...${cc_normal}"
+    wget --quiet --recursive --no-host-directories --cut-dirs=2 --no-parent --reject "index.html*" https://dl.bitmask.net/tuf/$TUF_ARCH/metadata.staged/
```
Comment (meskio) [inline tuf-stuff.sh]: The repo don't need to have the folder _metadata.staged_, I think we have it now, but better to fetch the folder _metadata_ and rename it as _metadata.staged_.

Soon there will be an unstable repo, this wget will need to be updated to fetch from unstable or stable depending which bundle you are generating.

**Comment by meskio** [comment]
A lot of bash black magic that I didn't know, but it looks good.

File: tuf-stuff.sh (hunk 1 above)
Comment (ivanalejandro0) [inline tuf-stuff.sh]: Ok, fixed the name of the folder.
I'll update the script when we have the unstable repo to contemplate that and also the release candidates.

**Comment by ivanalejandro0** [comment]
> A lot of bash black magic that I didn't know, but it looks good.

Ok.
If some of that magic looks too magical let me know and I can add some comment to clarify.

**Comment by ivanalejandro0** [comment]
Closed in favor of https://github.com/leapcode/bitmask_bundler/pull/1

**Pull request closed without merging**